operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -26,7 +27,12 @@ const (
 	// DefaultCentralDBPVCName is the default name for Central DB PVC
 	DefaultCentralDBPVCName = "central-db"
 
+	// DefaultCentralDBBackupPVCName is the default name for Central DB backup PVC
+	DefaultCentralDBBackupPVCName = "central-db-backup"
+
 	pvcTargetLabelKey = "target.pvc.stackrox.io"
+
+	defaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
```

Comment:
Note that `operator/internal/values/translation/translation.go` already defines `DefaultStorageClassAnnotationKey` and has some code which might be usable here, perhaps after some refactoring.

Reply:
Soo.. how about dropping this const? :-)
Interesting how this does not break the build, I always thought unused unexported identifiers are disallowed in Go 🤷🏻